### PR TITLE
feat : added card page with different variants

### PR DESCRIPTION
**Pull request by Kevin-Solomon**

### CODE REVIEW

File: pages/cards/cards.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
...%253EOur%2520changing%2520planet%253C%252Fh3%253E%250A%2520%2520%253Cp%2520class%253D%2522small-text%2522%253Eby%2520Kurt%2520Wagner%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522small-text%2522%253EVisit%2520ten%2520places%2520on%2520our%2520planet%2520that%2520are%2520undergoing%2520the%2520biggest%2520changes%2520today.%253C%252Fp%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-footer%2522%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520primary-btn%2522%253ERead%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-outline-primary%2522%253EBookmark%253C%252Fbutton%253E%250A%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
```
Comment (vp5h) [inline pages/cards/cards.html:72]: Is there a way you can minify this

**Review by vp5h — APPROVED** [review]

**Comment by Husain01** [comment]
Looks good overall. I guess the margin on the buttons isn't getting applied in cards with badges.

**Comment by Kevin-Solomon** [comment]
> Looks good overall. I guess the margin on the buttons isn't getting applied in cards with badges.

I will fix it asap.

**Comment by AnkurChunekar** [comment]
Hey kevin, the card components look great, IMO you can add more padding to the text only card and card with badge, it will look better. You can also make use of comments in the code. Rest everything is great keep going.

**Review by AnkurChunekar — APPROVED** [review]

**Comment by Kevin-Solomon** [comment]
> 

Thank you Ankur. I will make the suggested changes.

File: pages/cards/cards.html (hunk 1 above)
Comment (Kevin-Solomon) [reply pages/cards/cards.html:72]: I will get back to you. 
As far as I know,  I don't think there is a way.

**Review by kart-c — APPROVED** [review]
Looks good Kevin, although the `cards with badges` card, I think the margin bottom on the buttons is either missing or not applied

**Comment by Kevin-Solomon** [comment]
> Looks good overall. I guess the margin on the buttons isn't getting applied in cards with badges.

Hey husain , could you approve the code changes .

**Comment by Kevin-Solomon** [comment]
> Looks good overall. I guess the margin on the buttons isn't getting applied in cards with badges.

Hey husain , could you approve the code changes .

**Comment by Kevin-Solomon** [comment]
> 



> Looks good Kevin, although the `cards with badges` card, I think the margin bottom on the buttons is either missing or not applied

Done with the suggested changes.

**Comment by Kevin-Solomon** [comment]
> 



> Looks good Kevin, although the `cards with badges` card, I think the margin bottom on the buttons is either missing or not applied

Done with the suggested changes.

> Hey kevin, the card components look great, IMO you can add more padding to the text only card and card with badge, it will look better. You can also make use of comments in the code. Rest everything is great keep going.

Done with the suggested changes,Ankur.

**Review by kart-c — APPROVED** [review]
`cards.html` has some `img` tags without `alt` attributes. `cards.css` can use some refactor aswell, by removing the commented code if it is unnecessary. If it is intentionally kept then don't change it. Other than this, everything else looks good Kevin.

**Comment by Kevin-Solomon** [comment]
> `cards.html` has some `img` tags without `alt` attributes. `cards.css` can use some refactor aswell, by removing the commented code if it is unnecessary. If it is intentionally kept then don't change it. Other than this, everything else looks good Kevin.

Done with alt property. Thanks for the suggestion kartikeya .
This change helps keep my code consistent.

**Pull request merged**